roles/common/templates/etc-hosts.j2

```diff
@@ -3,7 +3,7 @@
 
 # Mesos mappings
 {% for item in groups.all %}
-{{item}} {{mesos_hostname}} {{hostvars[item]['ansible_hostname']}}
+{{item}} {{hostvars[item]['ansible_hostname']}}.{{ansible_dns}} {{hostvars[item]['ansible_hostname']}}
```

Comment:
Any reason why this reverts the change from d7a8d511d24b53f3ed693e711202ac7f68f3b5d3?